aten/src/THC/generic/THCTensorMathPointwise.cu

```diff
@@ -321,9 +322,6 @@ void THCTensor_(sigmoid)(THCState* state, THCTensor* self_, THCTensor* src) {
   }
 
   THCudaCheck(cudaGetLastError());
-#ifdef NAMEDTENSOR_ENABLED
```

Comment:
This is wrong; I think you intended to change NAMEDTENSOR_ENABLED to BUILD_NAMEDTENSOR.

Reply:
Right. I probably messed it up during conflict resolving.